lib/api/traversing.js

```diff
@@ -181,9 +181,9 @@ exports.parentsUntil = function (selector, filter) {
       }
     }, this);
 
-  return this._make(
```

Comment:
Updated this to match the pattern of most other functions.

Reply:
hmm, yes you are correct other traversing functions use filter function like that.